src/lib/structures/game/StandardGame.ts

```diff
@@ -40,10 +42,9 @@ export class StandardGame extends Game {
 
 	public async onTrackEnd() {
 		const everybodyPassed = this.round.passedPlayers.size === this.players.size;
-
 		const { tracksPlayed, playlistLength, nowPlaying } = this.queue;
 		let embedFooter = `${tracksPlayed}/${playlistLength}`;
-		let embedDescription = everybodyPassed ? 'Everybody passed! 🏃‍♂️' : 'Nobody got it! 🙁';
+		let embedDescription = everybodyPassed ? 'Everybody passed! 🏃' : 'Nobody got it! 🙁';
```

Comment:
Wait what changed here

Reply:
vscode was telling me there was some invisible character? I backspaced it and it seemed to change the man from blue to orange...Something to do with [person running](https://emojipedia.org/person-running/) vs [man running](https://emojipedia.org/man-running/)

Reply:
Oh that's just unicode being stupid. I put it at "person" originally to be gender neutral

Reply:
Nope it was man I'm pretty sure. (Man is blue, person is orange).

Reply:
Lol